packages/button-react/src/index.ts

```diff
@@ -1 +1 @@
-export { PrimaryButton, SecondaryButton, TertiaryButton } from "./Button";
+export { PrimaryButton, SecondaryButton, TertiaryButton, Props as ButtonPropsInterface } from "./Button";
```

Comment:
Du får dessverre ikke eksportert typer og interface direkte på denne måten. Tror du må først importere (for seg selv) og så eksportere dem, eller bare definere dem i ìndex`-fila..